manual/statements/built/README.md

```diff
@@ -1,5 +1,269 @@
+such as, `eq`, `ne`, `lt`, `in`, `contains`, `notNull`, etc.  In most cases, these
+functions receive 2 arguments: the name of the column, and the value to compare, but there
+are also variants that receive 2 iterables for columns and values correspondingly.
+
+Note: as queries are becoming more and more complex, repeating `QueryBuilder` at all
+places will make code less readable.  In this case to simplify the code you can import all
+(or only required) static functions of the `QueryBuilder` class (this is the same example
+from above):
+
+```java
+import static com.datastax.driver.core.querybuilder.QueryBuilder.*;
+
+//...
+
+BuiltStatement selectOne = QueryBuilder.select().from("test", "test")
+        .where(eq("id", 1));
+```
+
+In case if you need to specify complex conditions,  you can chain additional clauses
+together with the `and` operator, that accepts the same clauses as `where`:
+
+```java
+BuiltStatement select = QueryBuilder.select().from("test", "test")
+        .where(eq("id", "1")).and(eq("txt", "test"));
+```
+
+#### Other selection options
+
+For `SELECT` statements you can also specify a lot of different options: 
+ - `allowFiltering` generates a corresponding `ALLOW FILTERING` part of query (***only use if you know what you're doing!***);
+ - `limit` and `perPartitionLimit` allows to specify the amount of data to fetch;
+ - `groupBy` performs grouping of data;
+ - `orderBy` allows to specify sorting direction for specified clustering columns;
+ 
+This very "artificial" example shows the use for some of them:
+
+```java
+BuiltStatement selectOne = QueryBuilder.select().from("test")
+        .where(QueryBuilder.eq("id", 1)).limit(1).allowFiltering()
+        .perPartitionLimit(1).orderBy(desc("id"));
+```
+
+### Inserting data
+
+Insertion of data is straightforward--you specify the target table in call to
+`insertInto`, and then provide values to insert either by chaining several calls to the
+`value` function, or by using the `values` function and passing lists or arrays of column
+names and their corresponding values. The following 2 examples are equivalent:
+
+```java
+QueryBuilder.insertInto("test").value("id", 4).value("t", "test 4");
+QueryBuilder.insertInto("test").values(Arrays.asList("id", "t"), Arrays.asList(4, "test 4"));
+```
+
+You can also insert JSON-formatted data by calling the `json` function & passing the data:
+
+```java
+QueryBuilder.insertInto("test").json("{\"id\":4, \"t\":\"test 4\"}");
+```
+
+`QueryBuilder` also allows generation of the statement that use lightweight
+transactions (LWT) to check that inserted data doesn't exist yet.  You just need to add
+the call to `ifNotExists` to the statement:
+
+```java
+QueryBuilder.insertInto("test").value("id", 4).ifNotExists();
+```
+
+It also possible to specify additional metadata for inserted data, such as TTL (time to live) or
+timestamp.  This is achieved by using the `using` function and providing the `Using`
```

Comment:
Maybe you could say "achieved **with** the `using` method" to avoid the repetition.